src/app/screens/DetailScreen/index.tsx

```diff
@@ -4,12 +4,13 @@ import BookDetail from '@components/BookDetail';
 import { Book } from '@interfaces/book';
 import { Navigation } from '@interfaces/navigation';
 import CommentList from '@app/components/CommentsList';
+import { NAV_MARGIN } from '@constants/commonStyles';
 
 import styles from './styles';
 
 function DetailScreen({ route }: Navigation<Book>) {
   return (
-    <ScrollView style={styles.container}>
+    <ScrollView style={styles.container} contentContainerStyle={NAV_MARGIN}>
```

Comment:
This is wrong, you shouldn't have to add a margin for this. Instead, you can add a `SafeAreaView` to the screen:
```
<SafeAreaView ...>
  <ScrollView ...>
...
```

Reply:
tried that, but it doesn't get the size right, probably because i'm using a custom nav height. Or maybe I used it wrong 🤔